tests/test_due_diligence_verification.py

```
@@ -0,0 +1,469 @@
+                assert isinstance(doc["value"], (int, float, type(None))), f"Equipment value must be number or None, got {type(doc['value'])}"
+    
+    def test_stage_filtering_public(self, manager):
+        """Test that public stage hides file paths and only shows high-level stats."""
+        public_data = manager.get_stage_view("public")
+        
+        # Should have meta and high-level data
+        assert "meta" in public_data
+        assert "financials" in public_data
+        
+        # Should not have file paths in any documents
+        for category, data in public_data.items():
```

Ruff (0.12.2)

Loop control variable `category` not used within loop body

Rename unused `category` to `_category`

(B007)